docs/services/deploy/standalone.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Standalone"
+description: "Learn how to run standalone Restate services."
+---
+
+You can deploy your Restate service on any standalone machine.
+
+## Docker
+
+You can run your Restate service in a Docker container.
+
+Most of the Restate service templates come with a `Dockerfile` that you can use to build a Docker image for your service.
+
+## Running services behind a load balancer
+To spread load across multiple instances of services and higher availability, we recommend using a load balancer.
```

Comment:
Optionally, since we sometimes get asked:
```suggestion
To spread load across multiple instances of services and higher availability, we recommend using a load balancer. The Restate server does not currently support multiple endpoints for a single deployment.
```